keyvi/bin/keyvi_c/c_api.cpp

```diff
@@ -103,6 +103,16 @@ keyvi_dictionary* keyvi_create_dictionary(const char* filename) {
   }
 }
 
+keyvi_dictionary* keyvi_create_dictionary_with_loading_strategy(const char* filename,
+                                                                keyvi::dictionary::LoadingStrategy loading_strategy) {
+  try {
+    return new keyvi_dictionary(Dictionary(filename, loading_strategy));
+  } catch (const std::exception& e) {
+    std::cerr << e.what() << '\n';
```

Comment:
```suggestion
    std::cerr << e.what() << std::endl;
```

Reply:
clang-tidy says [avoid endl](https://clang.llvm.org/extra/clang-tidy/checks/performance/avoid-endl.html).

That's why the other one is already converted to use `\n`. So either I use `\n` or mark the line with `// NOLINT`